web/client/codechecker_client/cmd/cmd.py

```diff
@@ -1324,6 +1324,11 @@ def add_arguments_to_parser(parser):
   CC_CHANGED_FILES    Path of changed files json from Gerrit. Use it when
                       generating gerrit output.
 
+Exit status
+------------------------------------------------
+0 - Successful analysis
```

Comment:
Why the 1 error code is missing from here?
The diff command can exit with 1 error code in case of CodeChecker error:
https://github.com/Ericsson/codechecker/blob/1617b22cedcc76f0d5bf6e2a3fcc5cd58d1ca56f/web/client/codechecker_client/cmd_line_client.py#L815
https://github.com/Ericsson/codechecker/blob/1617b22cedcc76f0d5bf6e2a3fcc5cd58d1ca56f/web/client/codechecker_client/cmd_line_client.py#L819
https://github.com/Ericsson/codechecker/blob/1617b22cedcc76f0d5bf6e2a3fcc5cd58d1ca56f/web/client/codechecker_client/cmd_line_client.py#L1313

Don't forget to update the userguide too.